src/imitation/scripts/compare_to_baseline.py

```diff
@@ -0,0 +1,100 @@
+"""Compare experiment results to baseline results.
+
+This script compares experiment results to the results reported in the
```

Comment:
I wonder if it's better to compare two CSVs, and we just ship a "reference" CSV with the GitHub? That way we can allow things to deviate from the paper over time, which seems desirable (as we add new algorithms, environments). We could still include the paper results as a CSV to allow people to check for regressions over longer periods.

Reply:
I was thinking I'd just commit the latest results we have, whether that's from the paper or otherwise. We could include the paper results somewhere too although that feels less essential.

It's likely that many of the behaviors have changed since the paper so we may need to rerun to get up-to-date results (probably outside of this PR)